stubs/controller.py

```diff
@@ -157,6 +157,17 @@ def remove(self) -> None: ...
     def animationStartRecording(self, file: str) -> bool: ...
     def animationStopRecording(self) -> bool: ...
 
+    
+    def movieStartRecording(
+        self,
+        width: int,
+        height: int,
+        quality: int,
+        codec: int,
+        acceleration: int,
+        caption: bool) -> bool: ...
```

Comment:
Ah, sorry, I can see you've wrapped this to match the call site.
Since this is a stub file, it's more typical to just allow the longer lines and in this case I think the line is short enough not to worry about it?
```suggestion
    def movieStartRecording(self, width: int, height: int, quality: int, codec: int, acceleration: int, caption: bool) -> bool: ...
```

if that has a linter error I'd be tempted to add a `noqa` comment (with the error id).

Reply:
Hrm, this line is actually a lot wider than I thought. Wrapping is fine here.

Reply:
461b028a4700e09177a239b245bdbc8e90dc44c3